.github/workflows/ci.yml

```diff
@@ -2,9 +2,6 @@ name: CI
 
 on: push
 
-env:
-  NPM_TOKEN: ${{secrets.GA_TOKEN}}
-
```

Comment:
The `secrets.GA_TOKEN` is still referenced in this file (lines 123, 133, 143, 168, 178, 193), should the new `secrets.GH_PACKAGES_TOKEN` also be used there?

Reply:
Fixed all references except 178 and 193 because those belong to your Azure infrastructure that we’ll replace in #229.